pkg/node/uevent.go

```diff
@@ -418,3 +418,28 @@ func (handler *ueventHandler) processLoop(ctx context.Context) {
 		handler.processEvent(ctx, device, event["ACTION"])
 	}
 }
+
+func StartDynamicDriveDiscovery(ctx context.Context, identity, nodeID, rack, zone, region string, loopbackOnly bool) error {
+	handler := &ueventHandler{
+		nodeID: nodeID,
+		topology: map[string]string{
```

Comment:
do we need topology?

Reply:
yes, we need topology as uevent handler creates new drive objects if any drives are added